implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -446,3 +448,62 @@ pub(crate) fn replace_hyphen_with_stdin(s: String) -> String {
         s
     }
 }
+
+const EXIT_OK: i32 = 0;
+const EXIT_USAGE: i32 = 2;
+
+fn show_help(help: clap::Error) {
+    paginate(&help.render()).unwrap_or_else(|_| help.print().unwrap());
+
+    let _ = std::io::stdout().lock().flush();
+    let _ = std::io::stderr().lock().flush();
```

Comment:
It's taken from the `safe_exit` function from within `clap` and so I just copied it over. I think you're right in case the compiler is instructed to perform a proper unwind (is that always the case for Ockam?). If not, I suspect it may depend on whether there's additional (non-OS) buffering in place.